### PR TITLE
Made changes to files with bootstrap

### PR DESCRIPTION
**Pull request by joelnv**
Created validation states in Bootstrap and implement bootsrap to forms and post's view.

### CODE REVIEW

File: assets/styles.css [hunk 1]
```diff
@@ -1 +1,7 @@
-form > * {display:block;}
\ No newline at end of file
+form > * {display:block;}
```
Comment (aitchnyu) [inline assets/styles.css]: Why did you add this along with Bootstrap?

File: templates/base_layout.html [hunk 2]
```diff
@@ -4,11 +4,22 @@
 <html>
     <head>
         <link rel="stylesheet" type="text/css" href="{% static 'styles.css' %}"/>
```
Comment (aitchnyu) [inline templates/base_layout.html]: remove this too

File: templates/base_layout.html [hunk 3]
```diff
@@ -4,11 +4,22 @@
...
+        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
     </head>
-    <body>
+    <body STYLE="height:2000px;">
```
Comment (aitchnyu) [inline templates/base_layout.html]: Inline css should be minimised

File: templates/pform.html [hunk 4]
```diff
@@ -1,5 +1,5 @@
             {% csrf_token %}
-            <div class="form-group">
+            <div class="form-group {% if form.title.errors or form.non_field_errors%} has-error {% endif %}">
```
Comment (aitchnyu) [inline templates/pform.html]: Dont show this error for non field error

File: templates/base_layout.html (hunk 2 above)
Comment (joelnv) [reply templates/base_layout.html]: i am using extra css as i said

File: assets/styles.css (hunk 1 above)
Comment (joelnv) [reply assets/styles.css]: will create changes

File: templates/base_layout.html [hunk 5]
```diff
@@ -1,14 +1,20 @@
-{% load static from staticfiles %}
-{% load static %}
 <!DOCTYPE html>
 <html>
     <head>
-        <link rel="stylesheet" type="text/css" href="{% static 'styles.css' %}"/>
+        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
```
Comment (aitchnyu) [inline templates/base_layout.html]: Keep only one
